Makefile

```diff
@@ -2,9 +2,10 @@
 
 install:
 	@npm install
+	@cd example && npm install
 
 run:
-	@cd example && ../node_modules/.bin/webpack-dev-server --hot --inline --config ./webpack.config.js
```

Comment:
No, we use the main `webpack-dev-server` for the example to avoid reinstalling all the packages. Please revert this, and the changes in `example/package.json`.